db/migrate/20151109222454_create_vehicle.rb

```diff
@@ -0,0 +1,22 @@
+class CreateVehicle < ActiveRecord::Migration
+  def change
+    create_table :vehicles do |t|
+      t.string :brand, null: false
+      t.string :version, null: false
+      t.integer :maker_year, limit: 4, null: false
+      t.integer :model_year, limit: 4, null: false
+      t.string :license_plate, limit: 7, null: false
+      t.string :city
+      t.string :uf, limit: 2
+      t.string :color
+      t.string :fuel
+      t.string :renavam
+      t.string :chassi
+      t.string :owner
+      t.string :old_license_plate
+      t.string :status
```

Comment:
It is also a good idea to add uniqueness validation to `renavam` and `chassi`, is it not?